packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -321,14 +354,21 @@ export class ModularChangeFamily
 				!("revert" in change.change.valueChange),
 				0x4a9 /* Inverting inverse changes is currently not supported */,
 			);
+			assert(
+				path !== undefined,
+				0x4aa /* Only existing nodes can have their value restored */,
```

Comment:
I realize this was copy-pasted, and it _may_ be fine like this, but it's safer to use the literal string in the PR so the tool can re-index it.
```suggestion
				"Only existing nodes can have their value restored",
```

Reply:
Same for 0x477 below.